.gitignore

```diff
@@ -12,5 +12,4 @@ build/**
 cmake-build-debug/
 
 # ROM
-*.nds
```

Comment:
Why was this removed? If we're switching to naming the file .srl, this should be reflected in all other files that reference it.

Reply:
Duplicate entry